.github/actions/install-with-retries/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Install with retries'
+description: 'Runs yarn install with up to 2 retries'
+inputs:
+  skip-cypress-binary:
+    description: 'Whether to skip binary installation for cypress'
+    required: false
+    default: 'true'
```

Comment:
I find it weird that default can only be a string here. But I guess that's the way it is.

Reply:
You're correct, inputs are treated as strings and not boolean.

https://github.com/actions/runner/issues/1483

Reply:
Also: https://github.community/t/composite-actions-input-types-boolean-numeric/219254